### PR TITLE
#174 Log Analytics support

### LINKED ISSUE #174: Log Analytics support
As an engineer, I would like to create a function/cmdlet to support Log Analytics checks so that we can support future customers' potential needs.
	
- [x] Create `Confirm-LogAnalytics.ps1` and unit tests
- [x] Wire up `Confirm-LogAnalytics` to `Common.ps1`
- [x] Create example tests, bicep and documentation in `examples` folder

### PR DESCRIPTION
**Pull request by marclerwick**
Added Operational Insights Workspace cmdlets, tests, and examples.

Closes #174

### CODE REVIEW

File: examples/OperationalInsightsWorkspace/OperationalInsightsWorkspace_Example.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# How To Run OperationalInsightsWorkspace.Tests.ps1
+
+`OperationalInsightsWorkspace.Tests.ps1` contains examples of using the `Confirm-AzBPOperationalInsightsWorkspace`
+cmdlet.
+
+## Pre-Requisites
+
+- An Azure subscription to deploy resources to
```
Comment (msha1026) [inline examples/OperationalInsightsWorkspace/OperationalInsightsWorkspace_Example.md:8]: Add pre-req to have a resource group as well

File: Modules/BenchPress.Azure/Public/Confirm-OperationalInsightsWorkspace.ps1 [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-OperationalInsightsWorkspace {
+  <#
+    .SYNOPSIS
+      Confirms that an App Insights Workspace exists.
```
Comment (msha1026) [inline Modules/BenchPress.Azure/Public/Confirm-OperationalInsightsWorkspace.ps1:10]: App Insights -> Operational Insights

File: Modules/BenchPress.Azure/Public/Confirm-OperationalInsightsWorkspace.ps1 [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    .DESCRIPTION
+      The Confirm-AzBPOperationalInsightsWorkspace cmdlet gets an action group using the specified Action Group and
```
Comment (msha1026) [inline Modules/BenchPress.Azure/Public/Confirm-OperationalInsightsWorkspace.ps1:13]: action group/Action Group -> Operational Insights

**Review by msha1026 — APPROVED** [review]
Other than the few doc changes, looks good!

**Review by ozboms — APPROVED** [review]

**Pull request merged**